### DISCUSSION TITLE
Ability to add custom columns and change column names in Table view

### DISCUSSION THREAD

**Discussion by abel408** [Leantime/leantime#1631] [Ideas]
We are exploring Monday.com replacements. Leantime seems to check all the boxes, but it is important for us to have the ability to add custom column names in the table view of projects. Monday.com and Clickup.com both offer this as an option.

**Comment by marcelfolaron** [comment]
Hi, Thanks for your feature request. 
Custom fields are currently not possible but you can use tags which will autocomplete across a project. 
We may build a plugin for custom fields in the future but that is not on the immediate roadmap as of right now.
